### PR TITLE
Dev kobeya 754: Fixed Broken UI Skeleton File

### PR DESCRIPTION
**Pull request by Kobeyarai808**
- [x] Unit Testing
Nothing to visually inspect.

### CODE REVIEW

**Review by mhodgesatuh — APPROVED** [review]
Looks good overall.  
I have no idea what "live edit" means, which will likely be the case for newbies to this project.

**Comment by Kobeyarai808** [comment]
Live edit allows code changes to reflect instantly without restarting the spring app. I will figure out a place to document it.

File: myiam-overrides.skeleton.properties [hunk 1]
```diff
@@ -1,19 +1,12 @@
...
-# Fill in all blank values
 
 
-# these two should be your UH username
-groupings.api.test.admin_user= 
-groupings.api.localhost.user= 
+# this should be filled in if live edit is desired
 
 # location of your uh-groupings-ui project
 # example:   project.base-dir=file:c:/Users/me/Work/uh-groupings-ui
```
Comment (Kobeyarai808) [inline myiam-overrides.skeleton.properties:8]: Add examples for mac/linux

**Comment by devgav** [comment]
Looking at the most recent version of master on our UI it seems as though these changes have already been applied.

**Pull request closed without merging**